app/controllers/api_controller.rb

```diff
@@ -4,7 +4,8 @@ class ApiController < ActionController::API
   include Identifiable
 
   unless Rails.application.config.consider_all_requests_local
-    rescue_from ActiveRecord::RecordNotFound, with: -> { notfound }
+    rescue_from ActionController::ParameterMissing, with: -> { bad_request }
+    rescue_from ActiveRecord::RecordNotFound, with: -> { not_found }
     rescue_from CanCan::AccessDenied, with: -> { denied }
   end
```

Comment:
♻️ I think you're right about this block, I'm pretty sure we only get 500s from the app in production if these cases occur. I'll backlog it 👍